pkg/controller/unifiedpushserver/util.go

```diff
@@ -0,0 +1,55 @@
+package unifiedpushserver
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	aerogearv1alpha1 "github.com/aerogear/unifiedpush-operator/pkg/apis/aerogear/v1alpha1"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type image struct {
+	defaultImage string
```

Comment:
IMHO it could be a spec of the CR in order to allow we change it without the need to do a new build.

Reply:
I previously had this conversation earlier in this PR with @wei-lee, and we decided to go this way with env vars in the operator rather than a field in the CR: https://github.com/aerogear/unifiedpush-operator/pull/6/files/5d31078dfe82c3d95518f0ab301a3821c74e3513#discussion_r278063400